data/xml/2022.acl.xml

```diff
@@ -0,0 +1,1012 @@
+      <abstract>Translate-train is a general training approach to multilingual tasks. The key idea is to use the translator of the target language to generate training data to mitigate the gap between the source and target languages. However, its performance is often hampered by the artifacts in the translated texts (translationese). We discover that such artifacts have common patterns in different languages and can be modeled by deep learning, and subsequently propose an approach to conduct translate-train using Translationese Embracing the effect of Artifacts (TEA). TEA learns to mitigate such effect on the training data of a source language (whose original and translationese are both available), and applies the learned module to facilitate the inference on the target language. Extensive experiments on the multilingual QA dataset TyDiQA demonstrate that TEA outperforms strong baselines.</abstract>
+      <url hash="b15174aa">2022.acl-short.40</url>
+      <bibkey>yu-etal-2022-translate</bibkey>
+    </paper>
+    <paper id="41">
+      <title>C-MORE: Pretraining to Answer Open-Domain Questions by Consulting Millions of References</title>
+      <author><first>Xiang</first><last>Yue</last></author>
+      <author><first>Xiaoman</first><last>Pan</last></author>
+      <author><first>Wenlin</first><last>Yao</last></author>
+      <author><first>Dian</first><last>Yu</last></author>
+      <author><first>Dong</first><last>Yu</last></author>
+      <author><first>Jianshu</first><last>Chen</last></author>
+      <abstract>We consider the problem of pretraining a two-stage open-domain question answering (QA) system (retriever + reader) with strong transfer capabilities. The key challenge is how to construct a large amount of high-quality question-answer-context triplets without task-specific annotations. Specifically, the triplets should align well with downstream tasks by: (i) covering a wide range of domains (for open-domain applications), (ii) linking a question to its semantically relevant context with supporting evidence (for training the retriever), and (iii) identifying the correct answer in the context (for training the reader). Previous pretraining approaches generally fall short of one or more of these requirements. In this work, we automatically construct a large-scale corpus that meets all three criteria by consulting millions of references cited within Wikipedia. The well-aligned pretraining signals benefit both the retriever and the reader significantly. Our pretrained retriever leads to 2%-10% absolute gains in top-20 accuracy. And with our pretrained reader, the entire system improves by up to 4% in exact match.</abstract>
+      <url hash="899780b3">2022.acl-short.41</url>
+      <bibkey>yue-etal-2022-c</bibkey>
+    </paper>
+    <paper id="42">
+      <title>k-{R}ater {R}eliability: {T}he Correct Unit of Reliability for Aggregated Human Annotations</title>
+      <author><first>Ka</first><last>Wong</last></author>
```

Comment:
It looks like you're not applying case protection at all? You have to call `normalize()` on the title (see old script for an example).